### PR TITLE
fix(colour): use new dark theme colour

### PR DESCRIPTION
**Pull request by toolifelesstocode**
## Summary

The dark theme colour on the UI recently changed and so therefore this has changed `nextcord.Colour.dark_theme()` to use the new colour.

## Checklist

- [x] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
  - [ ] I have run `task pyright` and fixed the relevant issues.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: nextcord/colour.py [hunk 1]
```diff
@@ -277,12 +277,12 @@ def greyple(cls) -> Self:
 
     @classmethod
     def dark_theme(cls) -> Self:
-        """A factory method that returns a :class:`Colour` with a value of ``0x36393F``.
+        """A factory method that returns a :class:`Colour` with a value of ``0x313338``.
         This will appear transparent on Discord's dark theme.
 
         .. versionadded:: 1.5
```
Comment (ooliver1) [inline nextcord/colour.py:283]: Maybe this could benefit with a `.. versionchanged::`

**Review by ooliver1 — CHANGES_REQUESTED** [review]

**Review by teaishealthy — APPROVED** [review]
Just ooliver's comment, other than that lgtm :+1:

**Comment by ooliver1** [comment]
What benefit do you get from the old dark theme? You cannot access it anymore

**Comment by toolifelesstocode** [comment]
Not sure if I was suppose to use 'color' or 'colour' but I figured it doesn't matter much.

**Review by ooliver1 — APPROVED** [review]

**Pull request merged**